src/r2mm/installing/default_installation_rules/InstallationRuleApplicator.ts

```diff
@@ -166,6 +166,7 @@ export default class InstallationRuleApplicator {
             buildBepInExRules("DUSK"),
             buildBepInExRules("MyDreamSetup"),
             buildBepInExRules("MonsterTrain2"),
+            buildBepInExRules("ScheduleI"),
```

Comment:
This is not needed, and if it was, BepInExRules probably wouldn't work with MelonLoader game.